client/src/components/mapping/FhirMappingPanel/FhirResourceTree/secondaryLabel.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Attribute } from '@arkhn/fhir.ts';
+import { useSelector } from 'react-redux';
+import { Icon } from '@blueprintjs/core';
+
+import { IAttribute, IReduxStore } from 'types';
+
+interface SecondaryLabelProps {
+  attribute: Attribute;
+}
+
+export const SecondaryLabel = ({
+  attribute
+}: SecondaryLabelProps): React.ReactElement => {
+  const attributesForResource: { [k: string]: IAttribute } = useSelector(
+    (state: IReduxStore) => state.resourceInputs.attributesMap
+  );
+  const attributesWithInputs = Object.keys(attributesForResource).filter(
+    path => attributesForResource[path].inputs.length > 0
+  );
+
+  const checkHasChildInputs = (pathString: string) =>
+    attributesWithInputs.some(
+      el => el !== pathString && el.startsWith(pathString)
+    );
+
+  const checkHasInputs = (pathString: string) =>
+    attributesWithInputs.includes(pathString);
+
+  const isValidated = attributesForResource[attribute.path]?.comments.some(
+    c => c.validation
+  );
+  const comments = attributesForResource[attribute.path]?.comments.filter(
+    c => !c.validation
```

Comment:
I think it could be nice to be able to add a comment with a validation but maybe it changed to much the  logic you put in place

Reply:
yeah I thought about it and it's entirely compatible with the current datamodel (a validation comment may also have a `content`, but right now it's empty) but we will add that when we improve UX later it's that's alright with you